source/_includes/common-tasks/commandline.md

```diff
@@ -33,7 +33,7 @@ ha supervisor update
 
 ### Host
 
-```bash
+```txt
```

Comment:
These are valid shell commands and should therefore not be `txt` but `bash`. Please revert this change.

Reply:
Done. Reason was that the highlighting is kind of weird with the bash markdown style, but I see that this is probably better addressed somewhere else (in the markdown plugin).